cl/search/models.py

```diff
@@ -289,6 +289,10 @@ class Docket(AbstractDateTimeModel):
     SCRAPER_AND_HARVARD = 18
     RECAP_AND_SCRAPER_AND_HARVARD = 19
     HARVARD_AND_COLUMBIA = 20
+    IDB_AND_HARVARD = 24
+    RECAP_AND_IDB_AND_HARVARD = 25
+    SCRAPER_AND_IDB_AND_HARVARD = 26
+    RECAP_AND_SCRAPER_AND_IDB_AND_HARVARD = 27
     DIRECT_INPUT = 32
     DIRECT_INPUT_AND_HARVARD = 48
```

Comment:
It would be great to just identify the remaining options here now so we dont have to do this docket source dance anymore .

Reply:
Maybe just 21-to 23 and 28 to 31 to complete those scenarios

Reply:
I already identified all the required docket sources using the bulk data, we should no longer see the warning in the logs.